cli/Cargo.toml

```diff
@@ -77,7 +77,7 @@ birdcage = { version = "0.8.1" }
 
 [target.'cfg(windows)'.dependencies]
 # HACK: The Win32_Security feature is required by deno_io, but their crate didn't enable it
-windows-sys = { version = "0.52.0", features = ["Win32_Security"] }
+windows-sys = { version = "0.59.0", features = ["Win32_Security"] }
```

Comment:
Looks like this bump introduced some breaking changes.

Reply:
That's my bad for not reading the comment right above the line... I shouldn't have bumped this.